_projects/meta_review.md

```diff
@@ -0,0 +1,54 @@
+---
+collaborating_projects:
+  - coala
+desc: "Implement a meta-review system that tracks reactions on reviews."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - "https://github.com/coala/projects/issues/480"
+markdown: meta_review.md
+mentors:
+  - hemangsk
+  - blazeu
+  - alisianoi
+  - andrewda
+name: "Meta-review"
+requirements:
+  - "The applicant should have at least one patch accepted to any of the coala repositories."
+  - "The applicant should be familiar with GitHub GraphQL API and javascript."
+status: in_progress
+tags:
+  - community
+  - web
+---
+
+The goal of project meta-review is to build a meta-review track system similar to meta-moderation system that can track emoji responses to review comments. These responses would be collected, processed and displayed on [gh-board][https://github.com/coala/gh-board], which is a nice serverless kanban board.
+
+People including the author of pull request respond to comments by attaching emojis. Those emojis are called meta-review, or [reactions][https://developer.github.com/v4/enum/reactioncontent/]. THUMPS_UP and THUMPS_DOWN together with other reactions are used by people especially the author of pull request to publicly acknowledge or reject review comments. This provides feedback to reviewers and all other members.
```

Comment:
Just a thought, so github also provides confused, laugh, heart and tada emojis. We can use the ```confused``` to find out if the comments are descriptive enough. 

For example a common advice given to a newcomer PR is to follow coala guidelines for commit and amend the commit message. 

Now there are various ways in which reviewers can express this, some can be descriptive enough, some can be very descriptive with links and stuff, also some can be non descriptive comprising of our internal workflow lingo and hence non newcomer friendly. So maybe this is one of instances where we can exploit the confused emoji.